### ISSUE TITLE
About the lines 202-203 in stream_train.py

### ISSUE THREAD

**Issue by Zhou198** [njustkmg/TKDD21_S2OSC#1]
Hi there,

This paper is interesting and I appreciate your open source. But I have several confusions when I try to pair the code with the paper. 
https://github.com/njustkmg/TKDD21_S2OSC/blob/9a931985089e4d57c2f62903a19a850d2fd6d3df/stream_train.py#L202-L203
The above lines look like selecting observations from both in-class and out-of-class with higher probability scores (Line 202) and all observations from out-of-class (Line 203) as well, while equation (3) in the paper looks like referring to Line 202 only. All these observations (Lines 202 -203) will be used in the 2nd part of the below loss function (Line 214). However, **x_out_mask == True** contains those observations with **_novel_pseudo_label** from Line 159, which outbounds the labels in the below cross-entropy loss.

https://github.com/njustkmg/TKDD21_S2OSC/blob/9a931985089e4d57c2f62903a19a850d2fd6d3df/stream_train.py#L214

I really appreciate your time and help.
